src/test/java/api/loans/CheckOutCalculateDueDateTests.java

```diff
@@ -1085,4 +1085,28 @@ private DateTime currentYearDateTime(int month, int day, int hour, int minute,
       .withSecondOfMinute(second)
       .withMillisOfSecond(0);
   }
+
+  private JsonObject useFixedPolicy(
+    UUID fixedDueDateScheduleId, DueDateManagement dueDateManagement) {
```

Comment:
Please can this be formatted using the standard line splitting conventions
```suggestion
  private JsonObject useFixedPolicy(UUID fixedDueDateScheduleId, 
    DueDateManagement dueDateManagement) {
```